onnx/optimizer/passes/fuse_conv_add_into_bias.h

```diff
@@ -0,0 +1,43 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE { namespace optimization {
+
+struct FuseConvAddIntoBias final : public OptimizePass {
+  explicit FuseConvAddIntoBias()
+    : OptimizePass("fuse_conv_add_into_bias", API_TYPE::IR) {
+  }
+
+  static int idx_of_conv(const onnx::ArrayRef<onnx::Value *> & values) {
+    for (size_t i = 0; i < values.size(); i++)
+      if (values[i]->node()->kind() == kConv)
+        return i;
+    return -1;
+  }
+
+  void fuse_conv_add_into_bias(Graph& graph) {
+    for (auto it = graph.begin(); it != graph.end(); ++it) {
+      auto* n = *it;
+      if (n->kind() == kAdd) {
+        int idx = idx_of_conv(n->inputs());
+        if (idx != -1 && n->inputs()[idx]->node()->inputs().size() <= 2) {
+          auto origConv = n->inputs()[idx];
+          auto origBias = n->inputs()[1 - idx];
```

Comment:
You already have two input assumption here. :-)

Reply:
@houseroad 
Yes. And I don't know which input comes from conv op, so I use `idx_of_conv` to get the index.